html/pagelayout.html

```diff
@@ -67,15 +69,16 @@
     </style>
 </head>
 <body>
-    <div>
-    <header>
+    <div class="header">
+        <h2>HEADER </h2>
+    </div>
             
-    <nav>
+    <div class="nav">
         <a href="">home</a>
```

Comment:
Instead of having an empty href, you can give `href="/"`.

Reply:
done